resources/providers/awslib/ec2/provider.go

```diff
@@ -231,3 +240,16 @@ func (p *Provider) DescribeSnapshots(ctx context.Context, snapshot EBSSnapshot)
 	}
 	return result, nil
 }
+
+func (p *Provider) DeleteSnapshot(ctx context.Context, snapshot EBSSnapshot) error {
+	client, err := awslib.GetClient(aws.String(snapshot.Region), p.clients)
+	if err != nil {
+		return err
+	}
+	_, err = client.DeleteSnapshot(ctx, &ec2.DeleteSnapshotInput{SnapshotId: aws.String(snapshot.SnapshotId)})
+	if err != nil {
+		return fmt.Errorf("error deleting snapshot %s: %v", snapshot.SnapshotId, err)
```

Comment:
```suggestion
		return fmt.Errorf("error deleting snapshot %s: %w", snapshot.SnapshotId, err)
```

`%v` does not wrap errors: https://go.dev/play/p/JdJZ6ycbtF1

Reply:
I would suggest also updating the linter [settings](https://github.com/elastic/cloudbeat/blob/main/.golangci.yaml)